config/rbac/role.yaml

```diff
@@ -6,6 +6,26 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
```

Comment:
why changes here?

Reply:
I pulled one of Ashna's branches (to get eraser.go changes) and mistakenly committed that file. I already reversed it.